src/main/java/de/tum/in/www1/artemis/web/rest/lecture/AttachmentUnitResource.java

```diff
@@ -231,4 +284,32 @@ private void checkAttachmentUnitCourseAndLecture(AttachmentUnit attachmentUnit,
             throw new ConflictException("Requested lecture unit is not part of the specified lecture", "AttachmentUnit", "lectureIdMismatch");
         }
     }
+
+    /**
+     * Checks that the lecture exists and is part of a course, and that the user is at least editor in the course
+     *
+     * @param lectureId The id of the lecture
+     */
+    public void checkLecture(Long lectureId) {
```

Comment:
```suggestion
    private void checkLecture(Long lectureId) {
```
Otherwise put it in a service